src/mixxx.cpp

```diff
@@ -598,6 +596,11 @@ void MixxxMainWindow::initializeWindow() {
     Pal.setColor(QPalette::Background, MenuBarBackground);
     m_pMenuBar->setPalette(Pal);
 
+    // restore window size/position
+    QSettings settings("mixxxdj", "mixxx");
```

Comment:
To be honest I think it makes more sense to use QSettings here and have this kind of temporal state information stored separately from the user preferences. Why do you want to store it in mixxx.cfg? It's not a configuration option. 

It doesn't make sense to me to jump through a bunch of hoops to store the window state via your settingsmanager class when I can just do it the [standard QT way](http://doc.qt.io/qt-4.8/restoring-geometry.html) that everyone else uses.